.github/workflows/pythonpackage.yml

```diff
@@ -230,22 +239,17 @@ jobs:
       run: python -m pip install $PIP_FLAGS .
     - name: Check version
       run: |
-        # Interpolate version
-        if [[ "$GITHUB_REF" == refs/tags/* ]]; then
-          TAG=${GITHUB_REF##*/}
-        fi
-        THISVERSION=$( python get_version.py )
-        THISVERSION=${TAG:-$THISVERSION}
         INSTALLED_VERSION=$(python -c 'import niworkflows; print(niworkflows.__version__, end="")')
-        echo "VERSION: \"${THISVERSION}\""
         echo "INSTALLED: \"${INSTALLED_VERSION}\""
-        test "${INSTALLED_VERSION}" = "${THISVERSION}"
+        test "${INSTALLED_VERSION}" = "${VERSION}"
+      env:
+        VERSION: ${{ needs.job_metadata.outputs.version }}
     - name: Install test dependencies
       run: python -m pip install $PIP_FLAGS "niworkflows[tests]"
     - name: Run tests
       uses: GabrielBB/xvfb-action@v1
```

Comment:
Can this be removed now that we dropped surfplot?

Reply:
Let's try.